Bug: Compilation errors on Linux arm64

I'm test this library as a potential replacement for `libreoffice_rs` and I'm hitting compilation issues on `arm64`.

### Error message
Compilation fails on the function signature of `free_error` in `sys.rs`,  due to an expected `i8` value instead of `u8`.

```
   Compiling libreofficekit v0.4.0 (/src)
   error[E0308]: mismatched types
      --> src/sys.rs:460:25
          |
          460 |         self.free_error(raw_error);
              |              ---------- ^^^^^^^^^ expected `*mut i8`, found `*mut u8`
                  |              |
                      |              arguments to this method are incorrect
                          |
                              = note: expected raw pointer `*mut i8`
                                             found raw pointer `*mut u8`
                                             note: method defined here
                                                --> src/sys.rs:469:15
                                                    |
                                                    469 |     unsafe fn free_error(&self, error: *mut i8) {
                                                        |               ^^^^^^^^^^        --------------

error[E0308]: mismatched types
   --> src/sys.rs:472:24
       |
       472 |             free_error(error);
           |             ---------- ^^^^^ expected `*mut u8`, found `*mut i8`
               |             |
                   |             arguments to this function are incorrect
                       |
                           = note: expected raw pointer `*mut u8`
                                          found raw pointer `*mut i8`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `libreofficekit` (lib) due to 2 previous errors
```

The problem is similar to bugs previously addressed in [libreoffice-rs](https://github.com/undeflife/libreoffice-rs/issues/12).

### Context
I'm building `libreofficekit` (`git clone`) inside a Linux arm64 container image (via [qemu-user-static magic](https://github.com/multiarch/qemu-user-static)). 
- I have a Linux arm64 container image with rust installed and other libraries around
- I'm doing a `cargo build` of `libreofficekit` inside a Docker container

I was initially building `libreofficekit` as part of another app.

### Proposed changes
In the function signature of `free_error`, `i8` should be changed to `c_char`.
That will allow the compiler to decide whether or not this should translate into `i8` or `u8` depending on the underlying architecture.

--- a/src/sys.rs
+++ b/src/sys.rs
@@ -466,7 +466,7 @@ impl OfficeRaw {
     ///
     /// Used when we've obtained the error as we clone
     /// our own copy of the error
-    unsafe fn free_error(&self, error: *mut i8) {
+    unsafe fn free_error(&self, error: *mut c_char) {
         // Only available LibreOffice >=5.2
         if let Some(free_error) = (*self.class).freeError {
             free_error(error);

